### ISSUE TITLE
Exception when a file is not found

### ISSUE THREAD

**Issue by andreabedini** [envygeeks/jekyll-assets#488]
Caused by 9bba65d172324ef8c96f6d5a5d3ce6126333be45 which removed the `args` keyword from  `e_not_found` below.

https://github.com/envygeeks/jekyll-assets/blob/12084b8b2ac10dcacde363a10f98d6d9ca4d7e46/lib/jekyll/assets/tag.rb#L185-L190

https://github.com/envygeeks/jekyll-assets/blob/12084b8b2ac10dcacde363a10f98d6d9ca4d7e46/lib/jekyll/assets/tag.rb#L81-L85

Stack trace:

```
  Liquid Exception: unknown keyword: args in index.html
bundler: failed to load command: jekyll (/usr/local/bin/jekyll)
ArgumentError: unknown keyword: args
  /usr/local/lib/ruby/gems/2.5.0/gems/jekyll-assets-3.0.6/lib/jekyll/assets/tag.rb:185:in `e_not_found'
  /usr/local/lib/ruby/gems/2.5.0/gems/jekyll-assets-3.0.6/lib/jekyll/assets/tag.rb:82:in `rescue in render'
  /usr/local/lib/ruby/gems/2.5.0/gems/jekyll-assets-3.0.6/lib/jekyll/assets/tag.rb:69:in `render'
  /usr/local/lib/ruby/gems/2.5.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:109:in `render_node'
  /usr/local/lib/ruby/gems/2.5.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:88:in `block in render'
  /usr/local/lib/ruby/gems/2.5.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:75:in `each'
  /usr/local/lib/ruby/gems/2.5.0/gems/liquid-4.0.0/lib/liquid/block_body.rb:75:in `render'
```

**Comment by envygeeks** [comment]
This is known, however the commit is necessary as it was caused by stack optimization, and refactoring on my behalf, while missing something.  It was intentionally left errored for now because it pointed out a lapse in my testing.

It should be fixed in 3.0.7, thanks for creating a ticket that lets me close it directly.

**Comment by andreabedini** [comment]
Thanks for the prompt reply @envygeeks. I love the project!
